benchmark/clblas/clwrap.cpp

```diff
@@ -0,0 +1,331 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) 2021 Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename clwrap.cpp
+ *
+ **************************************************************************/
+
+#include "clwrap.h"
+
+#include <algorithm>
+#include <memory>
+
+char *getCLErrorString(cl_int err) {
```

Comment:
Where does this file come from? I thought the user would have to provide their own `clblas`?

Reply:
I'm not sure if I understood the question

Reply:
See https://github.com/codeplaysoftware/sycl-blas/pull/274 for a proposal to make a common library with this code

Reply:
Oh, I see, I thought this file came from `clblas`, that PR makes a lot of sense, thanks